docs/topics/azure-stack.md

```diff
@@ -170,9 +170,13 @@ The `Basic` load balancer SKU available on Azure Stack Hub limits the number of
 
 If you need to expose more than 5 services, then the recommendation is to route traffic to those services using an Ingress controller.
 
+### get-versions command
+
+The output of the `get-versions` command only pertains to Azure and not Azure Stack Hub clouds. The different upgrade paths can be found [here](https://docs.microsoft.com/azure-stack/user/azure-stack-kubernetes-aks-engine-upgrade#steps-to-upgrade-to-a-newer-kubernetes-version).
```

Comment:
Could we use this link to the version mapping table in the release notes? I will update this one provided to also point to the release notes: https://docs.microsoft.com/en-us/azure-stack/user/kubernetes-aks-engine-release-notes?view=azs-2005#kubernetes-version-upgrade-path-in-aks-engine-0510

Reply:
updated